src/compton.c

```diff
@@ -2019,7 +2019,8 @@ map_win(session_t *ps, Window id) {
 
   // Call XSelectInput() before reading properties so that no property
```

Comment:
Can you also update this comment?

Reply:
Sure, but what should I change it to? There is no `SelectInput` X11 request and I failed to come up with a good alternative to change this to.

Reply:
I was thinking something like "Set window event mask before reading properties"